src/lib.rs

```diff
@@ -731,6 +735,12 @@ impl Builder {
         self
     }
 
+    /// Set whether `Eq` should be derived by default.
+    pub fn derive_eq(mut self, doit: bool) -> Self {
+        self.options.derive_eq = doit;
```

Comment:
We can't compute `derive(Eq)` unless we're *also* doing `derive(PartialEq)`, so if `doit` is `true`, then this method should also set that option to `true`, and the documentation should describe this behavior.

And inside `derive_partial_eq()`, if its parameter is `false`, then it should also set `otions.derive_eq` to `false`, and its doc comment should describe that behavior too.